### PR TITLE
+25 Icons, +3 Links

### PR DESCRIPTION
**Pull request by ScaredPenguinXXL**
# Description

### Added 25 icons:
8Bit Photo Lab (`com.ilixa.ebp`)
Ticket Restaurant (`com.edenred.mobiletr`)
Apk Extractor (`com.ext.ui`)
Automatic Clicker (`simplehat.clicker`)
Ballz (`com.ketchapp.ballz`)
Dunk Shot (`com.ketchapp.dunkshot`)
Vectorify Da Home! (`com.iven.iconify`)
Mr Gun (`com.ketchapp.mrgun`)
Flipa Clip (`com.vblast.flipaclip`)
Bubble UPnP (`com.bubblesoft.android.bubbleupnp`)
Character Pad (`com.husseinelfeky.characterpad`)
Glitch Lab (`com.ilixa.glitch`)
Bluk (`com.pixelapestudios.bluk`)
Screen Stream (`info.dvkr.screenstream`)
Super Arcade Racing (`com.outofthebit.superarcaderacing`)
Photo & Picture Resizer (`com.simplemobilephotoresizer`)
Explorer (`org.amnh.explorer`)
Rider (`com.ketchapp.rider`)
Audio Converter (`com.bdroid.audiomediaconverter`)
Buildings For Minecraft PE (`com.electricfoal.buildingsformcpe`)
Crop And Trim Video (`fnzstudios.com.videocrop`)
Pocket Mathematics (`gecko.droid.mathematicshelper`)
Quick Cursor (`com.quickcursor`)
Word Sorting Machine (`wordsortingmachine.sbytex.com.wordsortingmachine`)
Guarda! (`it.yoomee.zanichelliapp`)

### Linked 3 icons:
Frequency Generator (`com.boedec.hoel.frequencygenerator` → `frequency_generator.svg`)
Voice Tools (`com.DevExtras.VoiceTools` → `voice_recorder.svg`)
Polette (`com.polette.shop.app` → `lenskart.svg`)

## Contributor's checklist

- [x] I followed [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: name="Bubble UPnP" → name="BubbleUPnP"

drawable="explorer" name="Explorer" →  drawable="explorer_amnh" name="Explorer AMNH"
explorer.svg → explorer_amnh.svg

name="Flipa Clip" → name="FlipaClip"

name="Frequency Generator" → name="Frequency Sound Generator"

name="Photo And Picture Resizer" → name="Photo &amp; Picture Resizer"

name="Screen Stream" → name="ScreenStream"

name="Ticket Restaurant" → name="Ticket Restaurant ~~ Edenred"

**Comment by x9136** [comment]
There are a couple of little things with alignment that I accepted as is so as not to waste our time. However, it remains to change the size of `pocket_mathematics.svg` and correct a couple of names in `appfilter.xml`.

**Comment by ScaredPenguinXXL** [comment]
Not sure why but Inkscape keeps telling me the width and height of the `pocket_mathematics.svg` is 160px * 160px. I redrew it from scratch though I can't guarantee it'll appear correctly. I have used svgo to optimize the icons beforehand in this pr, this time I'm simply exporting it as a plain.svg file.

Screenshot from 2024-04-07 12-29-48

Screenshot from 2024-04-07 12-32-16

Screenshot from 2024-04-07 12-33-56

**Comment by x9136** [comment]
> Not sure why but Inkscape keeps telling me the width and height of the pocket_mathematics.svg is 160px * 160px.

I have a hypothesis that this may be due to the peculiarities of creating ellipses and the fact that Inkscape works strangely with integer coordinates. I will add another option so that the size becomes as needed. If you are interested, then compare it with your own.
Screenshot 2024-04-07 at 14 01 39

**Pull request merged**